jaeger-jaxrs2/src/test/java/com/uber/jaeger/filters/jaxrs2/ClientFilterTest.java

```diff
@@ -63,8 +60,7 @@ public void setUp() throws Exception {
     tracer =
         new com.uber.jaeger.Tracer.Builder("Angry Machine", reporter, new ConstSampler(true))
             .build();
-    traceContext = new ScopeManagerTraceContext(tracer.scopeManager());
-    undertest = new ClientFilter(tracer, traceContext);
+    undertest = new ClientFilter(tracer);
```

Comment:
Doing this reduces code coverage because the deprecated methods are no longer exercised. Is there a way to exclude them from coverage?

Reply:
let me look around

Reply:
jacoco plugin version 0.8 seems to support this {[PR](https://github.com/jacoco/jacoco/pull/513)}

```
Methods annotated with @lombok.Generated (generated by Lombok getters, setters, equals, hashcode, toString, etc)
```

Reply:
yeah, that looks dirty. Why not just invoke the deprecated methods here, like `new ClientFilter(tracer, null);`? Strictly speaking just because a function is deprecated doesn't mean it does not need to work correctly, so exercising it in tests is a good thing.

Reply:
haha yes looks very wretched

yup! just gonna call the deprecate methods :)